File: csharp/examples/GetDocument.cs
using System;

namespace Examples.examples
{
    // Example of GET /documents/:document_id
    class GetDocument : BaseRequest
    {
        private string _documentId;

        public GetDocument() : base()
        {
            dotenv.net.DotEnv.Load();
            _documentId = Environment.GetEnvironmentVariable("DOCUMENT_ID");
            Console.WriteLine($"GetDocument: document_id={_documentId}");
        }

        public override void SendRequest()
        {
            string path = $"documents/{_documentId}";
            GetRequest(path);
        }
    }
}

File: php/examples/get_document.php
<?php
require_once 'base_request.php';

// Example of GET /documents/:document_id

$debug = true;
$document_id   = $_ENV['DOCUMENT_ID'];
$request_data = '';
$request_params = [];

send_request('GET', '/documents/' . $document_id, $request_params, $request_data);
